invenio_github/tasks.py

```diff
@@ -39,6 +40,9 @@
 from .models import Release, ReleaseStatus
 from .proxies import current_github
 
+remote = current_oauthclient.oauth.remote_apps['github']
```

Comment:
This has to go inside the task, otherwise you get classic "outside app context" error